widget/discrepancy-note/dn-widget.js

```diff
@@ -596,6 +594,23 @@ class Comment extends Widget {
 
     }
 
+    _isMapsReady( maps, counter ) {
+        let svg = maps[ counter ].querySelector( 'svg' );
+        setTimeout( () => {
```

Comment:
I wonder if timeout is still necessary with this code.

And I am also wondering if an entirely different approach would be better. Since you're looking at SVG elements existing, could you use a Mutationstracker instead of a loop?

Reply:
I tried to use MutationTracker (waitForQuietness) but it did not work as expected so I tried just add a new mutation observer one the new commit. Can you take a look? Thanks